src/app.ts

```diff
@@ -161,7 +162,23 @@ async function app() {
   await loadExternalDependencies(defs, config);
 
   // Load custom resources. In current tank configuration (input/fsh), resources will be in input/
+  await PluginManager.callHook(
+    'beforeLoadCustomResources',
+    path.join(input, '..'),
+    originalInput,
```

Comment:
Is it necessary to pass both the project directory _and_ the input directory?  Now that we've normalized to the recommended FHIR IG project structure, isn't it always the case that the inputs can be found at `${projectRoot}/input`?  Or are there exceptions that I'm not remembering?